<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opcionesLaser</class>
 <widget class="QDialog" name="opcionesLaser">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1543</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="OpLin">
     <property name="title">
      <string>Opciones de cálculo lineal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QPushButton" name="menuRutaEM2CW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="rutaEM1CW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>./EM1CW.txt</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="menuRutaEM1CW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ruta para CW desde EM1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ruta para CW desde EM2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rutaEM2CW">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>./EM2CW.txt</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="OpTerm">
     <property name="title">
      <string>Análisis térmico</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="3">
       <widget class="QGroupBox" name="Pump">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Fuente láser</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Radio haz tangencial [mm]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Radio haz sagital [mm]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="wPumpTan">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="wPumpSag">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Ángulo de divergencia [mrad]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="pumpDiv">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Longitud de onda [nm]</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="lambdaPump">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Potencia [W]</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="pumpP">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QGroupBox" name="Espejo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Espejo cóncavo</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Grosor [mm]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="tESpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Índice de refracción</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="nESpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="Lente">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Lente de enfoque</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="2">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Inclinación [°]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Radio de curvatura A [mm]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="inclinacionSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="R11Inf">
           <property name="text">
            <string>Infinito</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">R11</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QDoubleSpinBox" name="R11SpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="R11Fin">
           <property name="text">
            <string>Finito</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">R11</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Grosor [mm]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="tLSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Radio de curvatura B [mm]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="R12Inf">
           <property name="text">
            <string>Infinito</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">R12</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="R12Fin">
           <property name="text">
            <string>Finito</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">R12</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="R12SpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Índice de refracción</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="nLSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QGroupBox" name="Cristal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Datos del cristal</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Puntos de cálculo</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Tamaño Y [mm]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="tamX">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="N">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Tamaño X [mm]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="tamY">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Coef. absorción (alfa)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="alphaSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>N</zorder>
        <zorder>label_6</zorder>
        <zorder>label_7</zorder>
        <zorder>tamX</zorder>
        <zorder>label_8</zorder>
        <zorder>tamY</zorder>
        <zorder>label_27</zorder>
        <zorder>alphaSpinBox</zorder>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="enableTermico">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Realizar análisis térmico</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="Distancias">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Información de distancias</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Separación de fuente láser a lente [mm]</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="FuenteALenteSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Separación de lente a espejo [mm]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="LenteAEspejoSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="OpNoLin">
     <property name="title">
      <string>Opciones de cálculo no lineal</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Número de láminas para Ti:Zafiro</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>P. Intracavidad [kW]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Guardar en</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Guardar en</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="rutaEM2ML">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./EM2ML.txt</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Guardar en</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="enableEM2ML">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Cálculo desde EM2</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Umbral de error [%]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="numLaminasSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="rutaSpotIterEM1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./spotIteraciones/</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Iteraciones por ciclo</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="menuRutaSpotIterEM2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="rutaSpotIterEM2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="enableSpotIter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Tam. spot vs. iteración</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>EM2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="SpotIterMax">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Relación spot y núm. iteraciones.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="menuRutaEM2ML">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="menuRutaEM1ML">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="rutaIterMax">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="enableEM1ML">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Cálculo desde EM1</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="umbralSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="menuRutaIterMax">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="rutaEM1ML">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./EM1ML.txt</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>EM1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="menuRutaSpotIterEM1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</normaloff>../../../../../../usr/share/icons/Adwaita/22x22/places/folder.png</iconset>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="iterSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="PLaserSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rutaEM1CW</tabstop>
  <tabstop>menuRutaEM1CW</tabstop>
  <tabstop>rutaEM2CW</tabstop>
  <tabstop>menuRutaEM2CW</tabstop>
  <tabstop>enableEM1ML</tabstop>
  <tabstop>rutaEM1ML</tabstop>
  <tabstop>menuRutaEM1ML</tabstop>
  <tabstop>enableEM2ML</tabstop>
  <tabstop>rutaEM2ML</tabstop>
  <tabstop>menuRutaEM2ML</tabstop>
  <tabstop>enableSpotIter</tabstop>
  <tabstop>rutaSpotIterEM1</tabstop>
  <tabstop>menuRutaSpotIterEM1</tabstop>
  <tabstop>numLaminasSpinBox</tabstop>
  <tabstop>iterSpinBox</tabstop>
  <tabstop>umbralSpinBox</tabstop>
  <tabstop>enableTermico</tabstop>
  <tabstop>N</tabstop>
  <tabstop>tamX</tabstop>
  <tabstop>tamY</tabstop>
  <tabstop>FuenteALenteSpinBox</tabstop>
  <tabstop>LenteAEspejoSpinBox</tabstop>
  <tabstop>wPumpTan</tabstop>
  <tabstop>wPumpSag</tabstop>
  <tabstop>pumpDiv</tabstop>
  <tabstop>lambdaPump</tabstop>
  <tabstop>pumpP</tabstop>
  <tabstop>R11Inf</tabstop>
  <tabstop>R11Fin</tabstop>
  <tabstop>R11SpinBox</tabstop>
  <tabstop>R12Inf</tabstop>
  <tabstop>R12Fin</tabstop>
  <tabstop>R12SpinBox</tabstop>
  <tabstop>tLSpinBox</tabstop>
  <tabstop>inclinacionSpinBox</tabstop>
  <tabstop>nLSpinBox</tabstop>
  <tabstop>tESpinBox</tabstop>
  <tabstop>nESpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>opcionesLaser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>opcionesLaser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="R12"/>
  <buttongroup name="R11"/>
 </buttongroups>
</ui>
